.github/actions/docker-compose-app-e2e.yml

```diff
@@ -0,0 +1,18 @@
+services:
+  app:
+    container_name: "glpi-test-app-e2e"
+    image: "ghcr.io/glpi-project/${PHP_IMAGE:-githubactions-php-apache:8.3}"
+    environment:
+      CODE_COVERAGE: "${CODE_COVERAGE:-false}"
+      CI: true # Will be used to alter some configuration when running on github vs locally.
+      PLAYWRIGHT_BROWSERS_PATH: /opt/pw-browsers # Custom browser path to prevent some rights issues.
```

Comment:
Browsers should already be installed in the docker image and this variable should be set in the Dockerfile directly.